### DISCUSSION TITLE
How can I contribute to the plugin: peertube-plugin-livechat?

### DISCUSSION THREAD

**Discussion by ghost** [JohnXLivingston/peertube-plugin-livechat#169] [Q&A] [answered]
Hi, 

I would like to know how I can contribute to the plugin: peertube-plugin-livechat. How can I contribute to the plugin: peertube-plugin-livechat?

**Comment by JohnXLivingston** [comment, marked as answer]
Hello,

Contribution guidelines are here: https://johnxlivingston.github.io/peertube-plugin-livechat/contributing/
You can for example search for an open issue that has the tag «searching sponsor», and propose your help by commenting it: https://github.com/JohnXLivingston/peertube-plugin-livechat/issues?q=is%3Aissue+is%3Aopen+label%3A%22Searching+Sponsor%22
Note: I keep issues that don't have this tag for myself, as I have found sponsors for them.

Don't begin to work on an issue before I confirmed I'm ok with your proposal (to be sure there is no conflict with any work in progress)

**Reply by ghost** [reply]
Hello @JohnXLivingston, thanks for all information here!
